packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -28,13 +28,20 @@
   transition: all 0.2s;
 }
 
+.input:focus + .label:before {
+  box-shadow: 0 0 1px 0.125rem var(--color-outline);
+}
+
 .input:checked + .label::before {
   border-color: var(--color-green--dark);
   background-color: var(--color-green);
-  transition: all 0.2s;
 }
 
-.radioGroup {
-  display: inline-flex;
-  flex-direction: column;
+.input[disabled] + .label {
+  color: var(--color-blue--lighter);
```

Comment:
@rebecca-li I think to your comment, we should standardize this and rather than choosing from our full palette on every disabled element, introduce a `--color-disabled` that uses one of our colours. In [this proposal](https://www.figma.com/file/FLvj5SVGr0t40ErSf5hq4m/Design-System-Contribution-Colour-at-Jobber?node-id=30%3A581) I suggest a colour that's fairly close to `var--blue-lighter` for inactive states, so for this case (and going forwards) we could introduce the concept of an inactive colour to Atlantis similar to how we have an `error` colour at the moment.

Reply:
To my knowledge, we'd want to check Atlantis' `Checkbox`, `Button`, `Switch`, and possibly `Text` to see if there would be modifications needed there. I think in SG-1 we usually take that `60% or 70% of another grey` approach